src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeCache.targets

```diff
@@ -58,12 +168,11 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ComposeDir Condition="'$(ComposeDir)' != '' and '$(DoNotDecorateComposeDir)' != 'true'">$([System.IO.Path]::Combine($(ComposeDir), $(PlatformTarget)))</ComposeDir>
       <ComposeDir Condition="'$(ComposeDir)' == ''">$(DefaultComposeDir)</ComposeDir>
       <ComposeDir Condition="'$(DoNotDecorateComposeDir)' != 'true'">$([System.IO.Path]::Combine($(ComposeDir), $(_TFM)))</ComposeDir>
+      <CacheArtifactXml>$([System.IO.Path]::Combine($(ComposeDir),"artifact.xml"))</CacheArtifactXml>
```

Comment:
I am not picky with any of the names, suggestions are welcome. This is just the default name.
In fact users can rename the file when they are distributing it